src/XRFeitoriaUnreal/Content/Python/sequence.py

```diff
@@ -459,21 +459,22 @@ def add_fk_motion_to_binding(binding: unreal.SequencerBindingProxy, motion_data:
         )
     )
     rig_section: unreal.MovieSceneControlRigParameterSection = rig_track.get_section_to_key()
-    param_names = list(rig_section.get_parameter_names())
-    for bone_name, bone_data in motion_data[0].items():
-        if 'curve' in bone_data.keys():
-            bone_name = f'{bone_name}_CURVE_CONTROL'
-        else:
-            bone_name = f'{bone_name}_CONTROL'
-        assert bone_name in param_names, RuntimeError(f'bone name: {bone_name} not in param names: {param_names}')
+    # Note: assert bone_name is now achieved in function `xf_runner.SequenceUnreal._spawn_actor_in_engine`
+    # param_names = list(rig_section.get_parameter_names())
+    # for bone_name, bone_data in motion_data[0].items():
+    #     if 'curve' in bone_data.keys():
+    #         bone_name = f'{bone_name}_CURVE_CONTROL'
+    #     else:
+    #         bone_name = f'{bone_name}_CONTROL'
+    #     assert bone_name in param_names, RuntimeError(f'bone name: {bone_name} not in param names: {param_names}')
```

Comment:
Do you mean that the functionality of this code has been implemented in function `xf_runner.SequenceUnreal._spawn_actor_in_engine`? so whether this code should be removed?

Reply:
> [Unreal] Replace assert error with warning when adding FK motion. This allows you to add motions in an inaccurate way.

To achieve this, the assert of errors must be abandoned. Instead, the warning will be performed in:

https://github.com/openxrlab/xrfeitoria/blob/d9c48ba3e142ecc6dcbfce61e982dd6ce7bd992b/xrfeitoria/sequence/sequence_unreal.py#L298-L326
https://github.com/openxrlab/xrfeitoria/blob/d9c48ba3e142ecc6dcbfce61e982dd6ce7bd992b/xrfeitoria/sequence/sequence_unreal.py#L171
https://github.com/openxrlab/xrfeitoria/blob/d9c48ba3e142ecc6dcbfce61e982dd6ce7bd992b/xrfeitoria/sequence/sequence_unreal.py#L214